ironfish/src/common/constants.ts

```diff
@@ -0,0 +1,6 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+export const ENCRYPTED_NOTE_LENGTH = 32 + 32 + 32 + 83 + 16 + 64 + 16
+export const KEY_LENGTH = 32
+export const NOTE_LENGTH = 43 + 8 + 32 + 32
```

Comment:
I think these constants should live near the primitives that use them. `ENCRYPTED_NOTE_LENGTH` and `NOTE_LENGTH` should live in primitives where these numbers are derived from. `KEY_LENGTH` doesn't make sense because if I see this I have no idea which key this is the length of in our entire code base.

Reply:
Moved